### ISSUE TITLE
关于FGSD数据集训练的几点问题

### ISSUE THREAD

**Issue by 1667553930** [zf020114/CHPDet#7]
关于您在论文所述：”We used the Adam method [47] as the optimizer, and the initial learning rate was set to 2.5 × 10−4
. We trained our networkfor 140 epochs with a learning rate being dropped at 90epochs. During the training phase, we used random rotation, random flipping, and color jittering for data augmentation.“ 根据您的指示，我们将统一了训练模式，一种很简单但有效的优化方式，Adan优化，在90，120轮的时候进行一次学习率下调。但我们在训练数据处理时出现了问题。如您所述：" we used random rotation, random flipping, and color jittering for data augmentation." 在您的代码中，我们清晰的看见关于随机旋转它仅仅是一个小于1的参数，但您在获取仿射矩阵函数 get_affine_transform 中所需的旋转参数是一个角度而不是弧度。其次我认为我们之所以无法训练您的网络到一个好的精度表现，目前我们停留在81%左右的精度。可能是由于我们无法获取与您切割相同的数据集。在您公开的数据集中，需要通过步长820切成1024大小的图片。在您公开的切割代码中，有缩放比例[0.7,0.8,0.9,1,1.1]以及旋转比例，我们发现通过这样的切割方式获取的数据用来训练获得了较差的表现。我们诚挚的希望您能提供给我们一份关于FGSD训练集切割的代码，为了保证训练集一致。再次感谢您为科研事业做出的贡献。

**Comment by zf020114** [comment]
谢谢您的关注：
我已将代码发送给您
















在 2022-09-19 10:13:12，"1667553930" ***@***.***> 写道：

关于您在论文所述：”We used the Adam method [47] as the optimizer, and the initial learning rate was set to 2.5 × 10−4
. We trained our networkfor 140 epochs with a learning rate being dropped at 90epochs. During the training phase, we used random rotation, random flipping, and color jittering for data augmentation.“ 根据您的指示，我们将统一了训练模式，一种很简单但有效的优化方式，Adan优化，在90，120轮的时候进行一次学习率下调。但我们在训练数据处理时出现了问题。如您所述：" we used random rotation, random flipping, and color jittering for data augmentation." 在您的代码中，我们清晰的看见关于随机旋转它仅仅是一个小于1的参数，但您在获取仿射矩阵函数 get_affine_transform 中所需的旋转参数是一个角度而不是弧度。其次我认为我们之所以无法训练您的网络到一个好的精度表现，目前我们停留在81%左右的精度。可能是由于我们无法获取与您切割相同的数据集。在您公开的数据集中，需要通过步长820切成1024大小的图片。在您公开的切割代码中，有缩放比例[0.7,0.8,0.9,1,1.1]以及旋转比例，我们发现通过这样的切割方式获取的数据用来训练获得了较差的表现。我们诚挚的希望您能提供给我们一份关于FGSD训练集切割的代码，为了保证训练集一致。再次感谢您为科研事业做出的贡献。

**Comment by 1667553930** [comment]
谢谢您，这是我的邮箱1667553930@qq.com
